must-gather/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/kubernetes-csi/external-snapshotter/client/v8 v8.0.0
 	github.com/migtools/oadp-non-admin v0.0.0-20250324123917-741ae3b7a67d
 	github.com/openshift/api v0.0.0-20240912201240-0a8800162826
-	github.com/openshift/oadp-operator v1.0.2-0.20250327225822-5cc21cec9653
+	github.com/openshift/oadp-operator v1.0.2-0.20250506010707-2a52b7a20c4b
```

Comment:
How about we just do `replace ../` at the bottom?

See examples
https://go.dev/ref/mod#go-mod-file-replace

Reply:
Where are you saying it won't be fine?

Replace ../ should work here. Anywhere that `go get must-gather` will not be affected.. I guess that's where you would ideally want to continue updating the tag for those cases.

Reply:
What I meant was, must-gather is gonna be used/packaged as a standalone entity, so it might be better to rely on a tagged version, commit hash, or pseudo-version of oadp-operator rather than using a local replace directive. This ensures reproducibility, avoids local path dependencies in production builds, and makes it easier to manage and consume the module externally.

Reply:
https://github.com/kubernetes/kubernetes/blob/42634e39d385b0dda8a96bc1be8807df94b2fbd2/staging/src/k8s.io/client-go/go.mod#L67
Here's an example of what I mean.

Reply:
> [replace directives only apply in the main module’s go.mod file](https://go.dev/ref/mod#go-mod-file-replace:~:text=replace%20directives%20only%20apply%20in%20the%20main%20module%E2%80%99s%20go.mod%20file%20and%20are%20ignored%20in%20other%20modules.%20See%20Minimal%20version%20selection%20for%20details.) and are ignored in other modules. See [Minimal version selection](https://go.dev/ref/mod#minimal-version-selection) for details.

Reply:
Tho probably easier to keep track of changes this way ++